pkg/handlers/hpa_handlers.go

```diff
@@ -0,0 +1,78 @@
+package handlers
+
+import (
+	"net/http"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/BedrockStreaming/prescaling-exporter/pkg/services"
+	"github.com/BedrockStreaming/prescaling-exporter/pkg/utils"
+)
+
+// IHPAHandlers defines the interface for HPA handlers
+type IHPAHandlers interface {
+	CheckHPAMinimums(w http.ResponseWriter, r *http.Request)
+	GetPlatformScalingStatus(w http.ResponseWriter, r *http.Request)
+}
+
+// HPAHandlers implements IHPAHandlers
+type HPAHandlers struct {
+	hpaService services.IHPAService
+}
+
+// NewHPAHandlers creates a new instance of HPAHandlers
+func NewHPAHandlers(hpaService services.IHPAService) IHPAHandlers {
+	return &HPAHandlers{
+		hpaService: hpaService,
+	}
+}
+
+// CheckHPAMinimums
+// @Summary      Check if HPAs meet their required minimums
+// @Description  Checks if all HPAs in the cluster have at least the minimum number of running pods
+// @Tags         cluster
+// @Accept       json
+// @Produce      json
+// @Success      200  {object}  services.HPAMinimumStatus
+// @Router       /api/v1/hpas [get]
+func (h *HPAHandlers) CheckHPAMinimums(w http.ResponseWriter, r *http.Request) {
+	status, err := h.hpaService.CheckHPAMinimums()
+	if err != nil {
+		log.Error(err)
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	// If all HPAs don't meet their minimums, return a 200 OK status
+	if !status.AllHPAsMeetMinimum {
+		utils.WriteResponse(w, http.StatusOK, status)
+		return
+	}
+
+	utils.WriteResponse(w, http.StatusOK, status)
```

Comment:
```suggestion
	utils.WriteResponse(w, http.StatusOK, status)
```

same as https://github.com/BedrockStreaming/prescaling-exporter/pull/121/files#r2010590143